src/smokeTest/resources/application.conf

```diff
@@ -1,3 +1,4 @@
 test-url = "http://localhost:8583"
 test-url = ${?TEST_URL}
-
+test-pay-hub-url = ${TEST_PAY_HUB_URL}
+payments-queue-connection-string = ${PAYMENTS_QUEUE_READ_CONNECTION_STRING}
```

Comment:
would be best if `payments-queue-connection-string` contained 'read' imho.

Reply:
Renamed